x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -683,6 +683,13 @@ public static RoleDescriptor kibanaSystemRoleDescriptor(String name) {
                 // Fleet Server indices. Kibana create this indice before Fleet Server use them.
                 // Fleet Server indices. Kibana read and write to this indice to manage Elastic Agents
                 RoleDescriptor.IndicesPrivileges.builder().indices(".fleet*").allowRestrictedIndices(true).privileges("all").build(),
+                // Endpoint specific action responses. Kibana reads from these to display responses to the user.
+                RoleDescriptor.IndicesPrivileges.builder().indices(".logs-endpoint.action.responses-*").privileges("read").build(),
```

Comment:
I think we should merge this statement with the one from line 682:
https://github.com/elastic/elasticsearch/blob/3027039f72c26f6780cd7857a6c5c4b0c4bd0878/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java#L682

Read through the existing code, I'd prefer to put the merged statement _under_ the block starting at line 709 so that related grants stay together:
https://github.com/elastic/elasticsearch/blob/3027039f72c26f6780cd7857a6c5c4b0c4bd0878/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java#L709-L720

Reply:
statement is moved: https://github.com/elastic/elasticsearch/pull/81953/commits/f703c23577178435be56e269ec4afcd6d162ecd6